src/formatter-settings/assets/features/formatterSettings/components/Setting.tsx

```diff
@@ -28,28 +32,37 @@ const Setting = (): JSX.Element => {
     if (!value || value === "") {
       value = "0";
     }
-    dispatch(changeSetting({ id: id, value: value }));
+    onWillChangeSetting(id, value);
   };
 
   const handleSelect = (setting: JavaFormatterSetting, entry: string) => {
-    dispatch(changeSetting({ id: setting.id, value: entry }));
+    onWillChangeSetting(setting.id, entry);
   };
 
-  const generateSetting = (setting: JavaFormatterSetting) => {
+  const handleClick = (exampleKind: ExampleKind) => {
+    dispatch(activateExampleKind({ exampleKind: exampleKind }));
+  };
+
+  const generateSetting = (setting: JavaFormatterSetting): JSX.Element => {
     if (!setting.name || !setting.id || !setting.value) {
-      return;
+      return (<></>);
```

Comment:
Can we just return null here?

Reply:
Yeah, I find that `null` is a better practice from [Preventing Component from Rendering](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering).